npm/private/npm_translate_lock_generate.bzl

```diff
@@ -71,7 +71,7 @@ _FP_DIRECT_TARGET_TMPL = \
 
 _BZL_LIBRARY_TMPL = \
```

Comment:
I'd recommend [ignoring whitespace](https://github.com/aspect-build/rules_js/pull/1857/files?w=1) in the diff for this, it's mainly just putting things into a "for each alias" loop (which will normally have 1 alias and make no difference).